client/src/pages/Campaigns.js

```diff
@@ -1,11 +1,175 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+import MailIcon from '@material-ui/icons/Mail';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+import CustomizedButton from '../features/CustomizedButton';
 import NavBar from '../features/NavBar/MainBody';
+import UserInputContainer from '../features/UserInputContainer';
+import DataTable from '../features/DataTable';
+import { SampleData } from '../pages/sampledata2';
+
+import { getJWT } from '../utils';
+
+const useStyles = makeStyles((theme) => ({
+  createCampaignButton: {
+    width: 210,
+    fontSize: 14,
+  },
+  icon: {
+    margin: "16px 17px 0px 14px",
+  },
+  featuresContainer:{
+    padding: "100px 60px 30px",
+  },
+  campaignList: {
+    overflow: "auto",
+    width: "100%",
+    height: 600,
+    marginTop: 0,
+    [theme.breakpoints.down("lg")]: {
+      paddingLeft: 12,
+      paddingRight: 15,
+    },
+    [theme.breakpoints.down("md")]: {
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    [theme.breakpoints.down("sm")]: {
+      paddingLeft: 0,
+      paddingRight: 0,
+    }
+  }
+}));
+
+function CampaignDialog(props) {
+  const { open, onClose} = props;
```

Comment:
you can destructure the props on line 54
function CampaignDialog({open, onClose})